src/tasks/block.rs

```diff
@@ -62,6 +61,13 @@ impl InProgressBlock {
         self.transactions.push(tx.clone());
     }
 
+    /// Remove a transaction from the in-progress block.
+    pub fn remove_tx(&mut self, tx: &TxEnvelope) {
+        tracing::info!(hash = %tx.tx_hash(), "removing tx");
```

Comment:
a lot of these `info` seem too verbose. this seems like a `trace`

Reply:
gotcha, i was just going off of the same log level in `ingest_tx`. i can downgrade this to `trace`?

honestly in general i've questioned my instincts around what log levels to use.. any higher level guidance people have on this?

Reply:
https://github.com/init4tech/builder/pull/31/commits/b3d3d982707a5b87ec163b7e3a8e3d9c6dc0baf9